.rubocop.yml

```diff
@@ -1,12 +1,34 @@
 AllCops:
-  TargetRubyVersion: '2.2.0'
+  TargetRubyVersion: '2.7.0'
 
+# Naming section allows you to enforce consistent naming conventions across your codebase.
```

Comment:
Make this comment specific to the reason you're overriding the default behavior, not a description of the default. E.g. Answer the question **why** are you excluding `lib/omniauth-mlh.rb`?